### PR TITLE
fix: ``autoapi`` relative directory path wrt ``tox`` env

### LINKED ISSUE #486: Relative paths with `autoapi_dirs` failing in Tox environment
### Description

Extension error: AutoAPI directory `<path>\src\ansys` not found. Please check your `autoapi_dirs` setting.

It seems that relative paths configured for `autoapi_dirs` are failing when running in the Tox environment, resulting in errors related to the directory not being found.

### Tasks

- Verify the relative path with respect to different environments.
- Check and correct the relative path configuration.

### PR DESCRIPTION
**Pull request by Revathyvenugopal162**
resolves #486

### CODE REVIEW

File: src/ansys_sphinx_theme/extension/autoapi.py [hunk 1]
```diff
@@ -79,11 +80,11 @@ def prepare_jinja_env(jinja_env) -> None:
     app.config["autoapi_member_order"] = autoapi.get("member_order", "bysource")
 
     # HACK: The ``autoapi_dirs`` should be given as a relative path to the conf.py.
-    relative_autoapi_dir = os.path.relpath(
-        autoapi.get("directory", "src/ansys"), start=str(app.confdir / "conf.py")
-    )
-    app.config["autoapi_dirs"] = [relative_autoapi_dir]
-    print(f"AutoAPI directories: {app.config['autoapi_dirs']}")
+    autoapi_dir = autoapi.get("directory", "src/ansys")
+    doc_source_root_dir = Path(app.srcdir).resolve().parent.parent
```
Comment (RobPasMue) [inline src/ansys_sphinx_theme/extension/autoapi.py:84]: ``app.srcdir`` is "{ROOT}/doc/source", right? In that case, ``resolve().parent.parent`` would take you to {ROOT} but you are naming your variable ``doc_source_root_dir``. My point is, if this variable is going to be {ROOT}, let's just call it ``root_dir``. Please correct me if I'm wrong with the path resolution!

Comment (Revathyvenugopal162) [reply src/ansys_sphinx_theme/extension/autoapi.py:84]: You are right, we can call it ``root_dir``. Thanks Roberto

Comment (Revathyvenugopal162) [reply src/ansys_sphinx_theme/extension/autoapi.py]: updated in [31d850e](https://github.com/ansys/ansys-sphinx-theme/pull/494/commits/31d850e9e0a52d23fe19d9b74fe30a965950b8ff)

**Review by RobPasMue — APPROVED** [review]
LGTM! Thanks @Revathyvenugopal162

**Comment by Revathyvenugopal162** [comment]
i will do a patch release with this fix and https://github.com/ansys/ansys-sphinx-theme/pull/495.

**Pull request merged**